src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -51,6 +56,58 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
       return meUserInterestsLoader?.()
     },
   },
+
+  // moved InquirerCollectorProfileFields here
+  location: { type: myLocationType },
+  artsyUserSince: date,
+  ownerID: {
+    type: new GraphQLNonNull(GraphQLID),
+    description: "User ID of the collector profile's owner",
+    resolve: ({ owner: { id } }) => id,
+  },
+  icon: {
+    type: Image.type,
+    resolve: ({ icon }) => normalizeImageData(icon),
+  },
+  bio: {
+    type: GraphQLString,
+  },
+  profession: { type: GraphQLString },
+  otherRelevantPositions: {
+    type: GraphQLString,
+    description: "Collector's position with relevant institutions",
+    resolve: ({ other_relevant_positions }) => other_relevant_positions,
+  },
+  emailConfirmed: {
```

Comment:
(should these have the `is`-prefix for boolean fields?)

Reply:
I agree, but CX was already using the fields in that way, changing now would involve the deprecation of this field.
Q, is there an agreement over MP for boolean fields in this case to have `is` as a prefix? Not questioning your suggestion, I think it makes sense, just asking because I've seen lots of booleans without this prefix, would be nice at least here in MP to be consistent

Reply:
But this is a new field added to `CollectorProfile` type, so couldn't it be switched to use the preferred `is`-prefix? This is a new field.

Yes we definitely prefer the `is` prefix to Booleans and I could have sworn I documented this in a best practices doc, although I don't see it mentioned in https://github.com/artsy/README/blob/main/playbooks/graphql-schema-design.md

Reply:
I can open a follow-up PR tomorrow to deprecate it no big deal at all 👍 
It's faster if I do it than assigning it to the respective team and explaining the context 😂